ui/article-renderer/src/main/java/org/cru/godtools/article/fragment/ArticlesFragment.java

```diff
@@ -0,0 +1,95 @@
+import org.cru.godtools.article.adapter.ArticleAdapter;
+import org.cru.godtools.articles.aem.model.Article;
+import org.cru.godtools.articles.aem.view_model.ArticleViewModel;
+import org.cru.godtools.base.tool.fragment.BaseToolFragment;
+
+import java.util.Locale;
+import java.util.Objects;
+
+import butterknife.BindView;
+import timber.log.Timber;
+
+public class ArticlesFragment extends BaseToolFragment implements ArticleAdapter.Callback {
+    public static final String TAG = "ArticlesFragment";
+    private static final String MANIFEST_KEY = "manifest-key";
+
+    @Nullable
+    @BindView(R2.id.articles_recycler_view)
+    RecyclerView mArticlesView;
+
+    ArticleAdapter mAdapter;
+
+    public static ArticlesFragment newInstance(@NonNull final String code, @NonNull final Locale locale,
+                                               String manifestKey) {
+        ArticlesFragment fragment = new ArticlesFragment();
+        Bundle args = new Bundle();
+        populateArgs(args, code, locale);
+        args.putString(MANIFEST_KEY, manifestKey);
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
+                             @Nullable Bundle savedInstanceState) {
+        super.onCreateView(inflater, container, savedInstanceState);
+        return inflater.inflate(R.layout.fragment_articles, container, false);
+    }
+
+    @Override
+    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        setupArticleRecyclerView();
+    }
+
+    private void setupArticleRecyclerView() {
+        if (mArticlesView != null) {
+            mAdapter = new ArticleAdapter();
+            mAdapter.setCallbacks(this);
+            mAdapter.setToolManifest(mManifest);
+            DividerItemDecoration itemDecoration = new DividerItemDecoration(getContext(),
+                    DividerItemDecoration.VERTICAL);
+            itemDecoration.setDrawable(Objects.requireNonNull(getActivity()).getResources()
+                    .getDrawable(R.drawable.divider));
+
+            mArticlesView.addItemDecoration(itemDecoration);
+            mArticlesView.setLayoutManager(new LinearLayoutManager(getActivity()));
+            mArticlesView.setAdapter(mAdapter);
+
+            ArticleViewModel viewModel = ArticleViewModel.getInstance(getActivity());
+
+            String manifestKey = getArguments() != null ? getArguments().getString(MANIFEST_KEY) : "";
+
+            viewModel.getArticlesByManifest(manifestKey).observe(this, articles -> {
+                // This will be triggered by any change to the database
+                mAdapter.setArticles(articles);
+            });
+        }
+    }
+
+    @Override
+    public void onArticleSelected(Article article) {
+        Timber.d("You selected %s as your article", article.mTitle);
```

Comment:
yay for temporary debugging!

did you know that you can add a `tag` to the message with code like this:
```
Timber.tag("Articles Fragment").d("something happened!!");
```

obviously this isn't necessary for this temporary debugging message, but it is beneficial for permanent logging messages so you can track down log messages to component quicker.